src/ent/game.c

```diff
@@ -236,7 +239,7 @@ static inline void game_gen_alpha_cross_set(Game *game, int row, int col,
       board_get_word_edge(board, row, col - 1, WORD_DIRECTION_LEFT);
   const int right_col =
       board_get_word_edge(board, row, col + 1, WORD_DIRECTION_RIGHT);
-  int score = 0;
+  Equity score = 0;
```

Comment:
Not sure if we want to use EQUITY_ZERO_VALUE here. I think it's fine either way, so I'll leave it up to you.